prompt/prompt-executor/prompt-executor-clients/prompt-executor-mistralai-client/src/commonMain/kotlin/ai/koog/prompt/executor/clients/mistralai/MistralAIModels.kt

```diff
@@ -0,0 +1,25 @@
+package ai.koog.prompt.executor.clients.mistralai
+
+import ai.koog.prompt.executor.clients.LLModelDefinitions
+import ai.koog.prompt.executor.clients.mistralai.MistralAIModels.MISTRAL_MEDIUM_3_1
+import ai.koog.prompt.llm.LLMCapability
+import ai.koog.prompt.llm.LLMProvider
+import ai.koog.prompt.llm.LLModel
+
+public object MistralAIModels : LLModelDefinitions {
+
+    public val MISTRAL_MEDIUM_3_1: LLModel = LLModel(
+        provider = LLMProvider.MistralAI,
+        id = "mistral-medium-2508",
+        capabilities = listOf(
+            LLMCapability.Temperature,
+            LLMCapability.Completion,
+            LLMCapability.Tools
+        ),
+        contextLength = 131_072
+    )
+}
```

Comment:
Are you planning to add more models from here? https://mistral.ai/models
Otherwise, we can do it on our end later